src/docs/product/integrations/notification-incidents/discord/index.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Discord
+sidebar_order: 1
+description: "Learn more about Sentry's Discord integration and how you can get notified and manage issues right in Discord."
+---
+
+Get notified and manage issues right in Discord with Sentry's Discord integration.
+
+## Install
+
+<Note>
+
+Sentry owner, manager, or admin permissions are required to install this integration.
+
+</Note>
+
+1. In [sentry.io](https://sentry.io), nagivate to **Settings > Integrations > Discord**.
+
+1. Click "Add Installation".
+
+   ![Install Discord integration](discord-integration-details.png)
+
+1. A pop up window will open. If you're not logged in to Discord in your browser, you'll need to log in. If you're already logged in, you won't see this.
+
+   ![Log in to Discord](discord-login.png)
+
+1. Use the dropdown menu to select the Discord server you'd like to add Sentry to and press continue.
+
+   <Note>
+     You will need to have the "Manage Server" permission enabled for your
+     Discord account in the server you're trying to add the bot to.
+   </Note>
+
+   ![Select server](discord-server-select.png)
+
+1. Finally, once the server is selected, click authorize. The integration will be created and the Sentry Discord bot will join the selected server.
+
+Your Discord integration is now available to all projects in your Sentry organization.
+
+In the next section, we'll walk through configuring Discord notifications.
+
+## Configure
+
+Currently our Discord integration supports just issue alert rules, but metric alerts and personal notifications are planned for future updates to the integration.
+
+### Issue Alerts
+
+Set up an issue alert by going to **Alerts** and clicking "Create Alert". "Issues" is the default selection so you can just click "Set Conditions".
+
+![Create issue alert](discord-add-issue-alert.png)
+
+From here you can configure your issue alert conditions as desired. Discord is then added by selecting "Send a Discord notification" in the actions dropdown.
+
+Select the Discord server and channel you'd like to send the alert to. You can optionally specify tags here to include in the notification.
+
+![Discord issue alert action settings](discord-action-form.png)
+
+The channel field must be filled in with a Discord channel ID, _not_ a channel name (this is because multiple Discord channels can share a name). To get this channel ID, you must enable Discord's Developer Mode (Settings -> Advanced -> Developer Mode).
```

Comment:
We can exclude some of the finer implementation details. Also, include a link to Discord docs on how to enable developer mode.

```suggestion
The channel field must be filled in with a Discord channel ID, _not_ a channel name. To get this channel ID, enable Discord's Developer Mode (Settings -> Advanced -> Developer Mode).
```